File: Store/Store.BusinessLogicLayer/Models/PrintingEdition/PrintingEditionModel.cs
using Store.BusinessLogicLayer.Models.Author;
using Store.BusinessLogicLayer.Models.Base;
using Store.Shared.Enums;
using System.Collections.Generic;

namespace Store.BusinessLogicLayer.Models.PrintingEdition
{
    public class PrintingEditionModel : BaseModel
    {
        public string Title { get; set; }
        public string Description { get; set; }
        public decimal Price { get; set; }
        public Enums.PayStatus Status { get; set; }
        public Enums.EditionType Type { get; set; }
        public Enums.Currency Currency { get; set; }
        public List<AuthorModel> Authors { get; set; }
    }
}

File: Store/Store.Presentation/ClientApp/store-client/src/app/models/printing-edition.model.ts
import { CurrencyTypeEnum } from '../enums/currency-type.enum';
import { EditionType } from '../enums/edition-type.enum';
import { PayStatus } from '../enums/pay-status.enum';
import { AuthorModel } from './author.model';
export class PrintingEditionModel {
  title: string;
  description: string;
  price: number;
  status: PayStatus;
  editionType: EditionType;
  currency: CurrencyTypeEnum;
  authors: AuthorModel[];
}
